test/models/comment_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class CommentTest < ActiveSupport::TestCase
+   test "comment_on_post" do 
```

Comment:
I think we should remove this test, along with the `comment_on_post` method in the comment model before we merge this in.  Let's do these experiments with TDD via the lessons on a branch other than master.